File: Unidad3/C#/u3p17.cs
using System;

class Program
{
    static void Main()
    {
        char opcion;
        do
        {
            Console.WriteLine("Bienvenido  a una sola ejecucion");
            Console.WriteLine("Escribe n para salir!!");
            opcion = Convert.ToChar(Console.ReadLine());
        } while (opcion != 'n');
    }
}

File: Unidad3/Java/U3p17.java
package com.mycompany.u3p17;
import java.util.Scanner;

public class U3p17 {

    public static void main(String[] args) {
        Scanner xd = new Scanner(System.in);
        char opcion;
        do{
            System.out.println("Bienvenido  a una sola ejecucion");
            System.out.println("Escribe n para salir!!");
            opcion=xd.next().charAt(0);
        }while(opcion != 'n');
        }
}
